CHANGELOG.md

```diff
@@ -1,109 +1,99 @@
-## 0.10.x
+# 0.10.x
```

Comment:
I guess it does look better even though I think there's never supposed to be >1 h1 on a page :)